### PR TITLE
Reduce yaml file for crypt_no_lvm

### PR DESCRIPTION
**Pull request by paolostivanin**
Ticket: https://progress.opensuse.org/issues/155869

MR: https://gitlab.suse.de/qe-yam/openqa-job-groups/-/merge_requests/212

VRs: https://openqa.suse.de/tests/14431116 + https://openqa.suse.de/tests/14431117

### CODE REVIEW

File: schedule/yam/test_cases/crypt_no_lvm.yaml [hunk 1]
```diff
@@ -7,7 +7,6 @@ description: >
   Encrypted installations can take longer, especially on non-x86_64
   architectures.
 vars:
-  DESKTOP: gnome
   ENCRYPT: 1
```
Comment (jknphy) [inline schedule/yam/test_cases/crypt_no_lvm.yaml:10]: let's move all the settings to job groups.

**Comment by jknphy** [comment]
Please always reference the MR in the PR as well, so the code review can be more consistent.

**Review by rakoenig — APPROVED** [review]
LGTM.

**Comment by tinawang123** [comment]
LGTM

**Pull request merged**